tianshou/highlevel/experiment.py

```diff
@@ -293,7 +297,7 @@ def run(
                 self._watch_agent(
                     self.config.watch_num_episodes,
                     policy,
-                    self.env_factory,
+                    watch_env,
```

Comment:
Let's use kwargs here pls

Reply:
Why? It's an internal function where all args have different types. And its only application uses clearly named variables for all args. Keyword arguments would only repeat the same names for no reason.